charts/spinnaker/templates/configmap/halyard-config.yaml

```
@@ -132,8 +133,16 @@ data:
                 {{ if $account.readPermissions }}--read-permissions {{ join " --read-permissions " $account.readPermissions }}{{ end }} \
                 {{ if $account.writePermissions }}--write-permissions {{ join " --write-permissions " $account.writePermissions }}{{ end }} \
                 --provider-version v2
+    EOF
     {{- end }}
 
+    # Run the commands in parallel over the account names limited by the specified parallelism factor.
+    xargs -P {{ .Values.halyard.installParallelism }} -I {} sh -c 'eval "$1"' - {} <<'EOF'
+    {{- range $name, $account := .Values.kubeConfig.accounts }}
+    source /tmp/upsert_{{ regexReplaceAll "\\.|-" $name "_" }}
```

Could this be simplified to:

```suggestion
    xargs -P {{ .Values.halyard.installParallelism }} -I {} sh {} <<'EOF'
    {{- range $name, $account := .Values.kubeConfig.accounts }}
    /tmp/upsert_{{ regexReplaceAll "\\.|-" $name "_" }}
```

---

Thought it wouldn't work due to permissions (file won't have the executable bit), but think it might work since `source`ing also "executes" it. Let's try. 👍